helix-core/src/test/java/org/apache/helix/tools/TestClusterVerifier.java

```diff
@@ -132,8 +132,14 @@ public void testDisablePartitionAndStopInstance() throws InterruptedException {
     // Disable partition for 1 instance, then Full-Auto ExternalView should not match IdealState.
     _admin.enablePartition(false, _clusterName, _participants[0].getInstanceName(), FULL_AUTO_RESOURCES[0],
         Lists.newArrayList(FULL_AUTO_RESOURCES[0] + "_0"));
-    Thread.sleep(1000);
-    Assert.assertFalse(strictMatchVerifier.verify(3000));
+
```

Comment:
Please refer to my change in the wagedRebalancer branch, using the bestPossibleVerifier directly should help to solve the problem.
Although I'm not sure if that is also related to what I have changed, could you please try that one?

Reply:
Thanks for the comment. I removed the Thread.sleep and change it with BestPossibleExternalViewVerifier to make sure resources have converged. As we discussed, I also add BestPossibleExternalViewVerifier next to StrictMatchExternalViewVerifier to make sure test functionality has not been changed. Thanks.